etc/spack/defaults/packages.yaml

```diff
@@ -19,15 +19,16 @@ packages:
     providers:
       awk: [gawk]
       blas: [openblas]
-      daal: [intel-parallel-studio+daal]
+      daal: [intel-daal]
       elf: [elfutils]
       golang: [gcc]
-      ipp: [intel-parallel-studio+ipp]
+      ipp: [intel-ipp]
       lapack: [openblas]
-      mkl: [intel-parallel-studio+mkl]
+      mkl: [intel-mkl]
```

Comment:
We should never default to `intel-parallel-studio`. It requires a license, whereas all of these other standalone libraries don't seem to require a license anymore. Plus we don't want to pollute our RPATH with non-MKL things.